lib/api/protocol/elementIdDisplayed.js

```diff
@@ -13,6 +13,10 @@ module.exports = class Session extends ProtocolAction {
   async command(webElementId, callback) {
     ProtocolAction.validateElementId(webElementId, 'elementIdDisplayed');
 
+    if (this.api.isAppiumClient()) {
```

Comment:
I was trying that before, but because `isElementDisplayed` is an async function, it will always return a promise and so if we try to return the `/displayed` endpoint from `isElementDisplayed`, it won't work as [this](https://github.com/nightwatchjs/nightwatch/blob/58ce1be856aaaa2a80074495f1d6485c98f44669/lib/transport/selenium-webdriver/actions.js#L52-L64) block of code would still be skipped.

I also thought of doing the skipped bit in `isElementDisplayed` but `sessionId` is not available in method-mappings.

Reply:
One approach to this I just thought of now can be to remove the `async` from `isElementDisplayed` in method-mappings and just return `element.isDisplayed()` which in itself is a promise? So, `isElementDisplayed` in method-mappings would look like:

```js
isElementDisplayed(webElementOrId) {
  if (this.transport.api.isAppiumClient()) {
    return `/element/${webElementOrId}/displayed`;
  }

  const element = this.getWebElement(webElementOrId);

  return element.isDisplayed();
},
```

The only thing with this approach is that for all other WebElement related commands, we first await the `element.isProperty()` and then return it, so this will be the only one different. But both the writing style should wok the same.

![image](https://user-images.githubusercontent.com/39924567/213439158-b8c9ab89-8a67-4250-8d86-1c0ed2471728.png)

Reply:
yeah, you can try that.